server/build.gradle

```diff
@@ -304,21 +304,6 @@ thirdPartyAudit.excludes = [
   'com.google.common.geometry.S2LatLng',
 ]
 
-if (project.runtimeJavaVersion <= JavaVersion.VERSION_1_8) {
```

Comment:
Won't runtime java still switch between java 8 and 10/11, so we would still need these?

Reply:
Runtime will still switch to 8, but  we no longer need to put in exceptions when running on 8. 
The reasons we had these exceptions is that  some third party dependencies like log4j have multi release jars. When we extract these jars, we get the class files for java 9 as well, and since the scan picks up all class files, we got to scan them, and got missing class errors when running on java 8 for classes that were added lather. With this change, we no longer extract class files meant for java 9 , 10, 11 in multi release jars when running on 8, so we won't run into this problem.